CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Changed
+
+- change temp directory to `.enhance/` from `tmp/`
```

Comment:
My only comment is one based partially on ignorance. Do plugins run in parallel in Architect? Could two plugins be operating on the `.enhance` folder at the same time? It might be better for each plugin to have a sub-directory under `.enhance` like `.enhance/arc-plugin-styles`.

Reply:
This is good input. 
I am thinking that we put this patch out now then move the creation / management of the `.enhance` folder up into `arc-plugin-enhance` in the near future.